packages/styles/scss/themes/expressive/components/_inline-notifications.scss

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '@carbon/layout/scss/convert';
+//-----------------------------
+// Inline Notifications  (expressive)
+//-----------------------------
+/// Inline Notifications styles (expressive)
+/// @access private
+/// @group inline-notifications-expressive
+@mixin inline-notifications-expressive {
+  .#{$prefix}--inline-notification {
+    .#{$prefix}--inline-notification__action-button.#{$prefix}--btn--ghost {
+      padding: calc(0.375rem - 3px) 1rem;
+    }
+    .#{$prefix}--inline-notification__title {
+      @include type-style('productive-heading-01');
```

Comment:
So we probably don't want to set type styles here since they should already be set from Jeff's type tokens (`_tokens.scss`). You may have to do some digging to see why the font type does not match up

Reply:
Looking at the ticket, I don't see anything about the font type being incorrect - do we need to change the title and subtitle?